modules/renter/contractor/dependencies.go

```diff
@@ -62,14 +62,19 @@ type (
 	}
 )
 
-// Because wallet is not directly compatible with modules.Wallet (wrong
-// type signature for StartTransaction), we must provide a bridge type.
-type walletBridge struct {
-	w walletShim
+// WalletBridge is a bridge for the wallet because wallet is not directly
+// compatible with modules.Wallet (wrong type signature for StartTransaction),
+// we must provide a bridge type.
+type WalletBridge struct {
+	W walletShim
 }
 
-func (ws *walletBridge) NextAddress() (types.UnlockConditions, error) { return ws.w.NextAddress() }
-func (ws *walletBridge) StartTransaction() transactionBuilder         { return ws.w.StartTransaction() }
+// NextAddress computes and returns the next address of the wallet.
+func (ws *WalletBridge) NextAddress() (types.UnlockConditions, error) { return ws.W.NextAddress() }
+
+// StartTransaction creates a new transactionBuilder that can be used to create
+// and sign a transaction.
+func (ws *WalletBridge) StartTransaction() transactionBuilder { return ws.W.StartTransaction() }
```

Comment:
revisiting this later, I think it's preferable for `StartTransaction` to just return a `modules.TransactionBuilder`. It doesn't make the mocking that much harder, and in doing so we can avoid the ugly `WalletBridge` code.